Assets/Scripts/StandardSamples/UI/Storage/Title/UITitleStorageMenu.cs

```diff
@@ -49,23 +49,6 @@ public class UITitleStorageMenu : SampleMenu
         public Text FileContent;
         private List<string> CurrentTags = new List<string>();
 
-        protected override void Awake()
```

Comment:
This was removed because with the new config system it is not possible to get into a situation where the encryption key is invalid - unless the user is trying _really_, _really_ hard. You basically need to edit the json config file manually. And when the encryption key is invalid, it will log an error when it comes time to use it instead of checking pre-emptively. I felt like that would be a more appropriate place to alert the user since it will be closely attached to its usage.

In the old system, there is a button to "generate" an encryption key, in the new system it's just created for you and there is not a manual way to alter it. Still works with backwards compatibility because users' old configuration files will be migrated, so the old encryption key they may have manually defined will still be present.

The details of that change are a little out of scope for _this_ PR, but the simplest change was to make sure that the encryption key invalidity is mentioned right when it's created, and then if it's still bad only when it's used.